io_scene_godot/converters/armature.py

```diff
@@ -85,7 +85,10 @@ def generate_bones_mapping(export_settings, armature_obj):
     """Return a dict mapping blender bone name to godot bone id"""
     bone_id = 0
     bones_mapping = dict()
-    for pose_bone in armature_obj.pose.bones:
+    sorted_keys = {k: v for k, v in sorted(armature_obj.pose.bones.items(), key=lambda item: - len(item[1].children))}
```

Comment:
Thanks for trying to address the issue.
Are you saying that the armature fails to be imported to Godot because Godot always expect parent pone comes before the child?  This only sorts the bones by the number of children they have. It does not ensure the parent-before-child order. I think topology sort on the `armature_obj.pose.bones` is one of the approaches to fix this.